neuro-sdk/src/neuro_sdk/_buckets.py

```diff
@@ -228,13 +233,15 @@ async def create(
         self,
         name: Optional[str] = None,
         cluster_name: Optional[str] = None,
-        org_name: Optional[str] = None,
+        org_name: Union[Optional[str], OrgNameSentinel] = ORG_NAME_SENTINEL,
```

Comment:
Should the documentation be updated?

Also, you can use `Union[str, None, Literal[ORG_NAME_SENTINEL]]` as a type annotation.

Reply:
Hmm, probably I missing something. `Literal` generic can only work with literal types (like ints, strings, bools, None) and enums. But `OrgNameSentinel` is just ordinary class. Do you suggest it to replace with an enum?

Reply:
Correct. There is a draft PEP for singletons, it is not implemented AFAIK